release/models/system/openconfig-system-bootz.yang

```diff
@@ -0,0 +1,100 @@
+module openconfig-system-bootz {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/system-bootz";
+  prefix "oc-sys-bootz";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-inet-types { prefix oc-inet; }
+  import openconfig-network-instance { prefix oc-ni; }
+  import openconfig-system { prefix oc-sys; }
+  import openconfig-types { prefix oc-types; }
+  import openconfig-yang-types { prefix oc-yang; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module adds configuration and operational state relating to bootz
+    service running on a network device.";
+
+
+  oc-ext:openconfig-version "1.0.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision "2023-06-16" {
+    description
+      "Creation of bootz state paths needed for the service.";
+    reference "1.0.0";
+  }
+
+  grouping bootz-service-structural {
+    description
+      "Structural grouping for bootz service that can be enabled on
+      the system.";
+
+    container state {
+        config false;
+        description
+        "Operational state relating to the bootz service.";
+          uses bootz-state;
+    }
+  }
+
+  grouping bootz-state {
+    description
+      "State parameters required to monitor bootz service.";
+
+    leaf checksum {
+      type string;
+      default "";
+      description
+        "The current checksum of the bootz protocol buffer.
```

Comment:
which message type from the [bootz.proto@0d91289e](https://github.com/openconfig/bootz/blob/0d91289e33f8c10a46373fdf8c0e81d208f24849/proto/bootz.proto) is this referencing to?